site_scons/buildutils.py

```diff
@@ -10,20 +10,22 @@
 import enum
 from pathlib import Path
 import logging
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Union, List, Dict, Generator
```

Comment:
If you quote these when they're used, it functions as a forward declaration for typing purposes, but it isn't evaluated at runtime.